docs/lro-with-original-url-as-final-state.md

```diff
@@ -0,0 +1,13 @@
+# LroWithOriginalUriAsFinalState
+
+## Description
+
+The long running operation with final-state-via:original-uri should have a sibling 'get' operation.
+
+## How to fix
+
+Add the missing get operation.
+
+## What's the impact if breaking the rule
+
+If no get operation, the generated SDK could not know the response schema of the final state.
```

Comment:
If get operation is missing, SDK generation may fail for the response schema of the final state is not available.

Reply:
seem a little confusion for me, actually I can tell more here, if no get operation, SDK have no way to determine the response schema of the final state  , as it 's same with get operation schema